metadata-ingestion/tests/integration/airbyte/golden_test_airbyte_oss_ingest.json

```diff
@@ -0,0 +1,791 @@
+[
+{
+    "entityType": "dataFlow",
+    "entityUrn": "urn:li:dataFlow:(airbyte,2de688a6-94ee-463a-949b-5a8e808641c0,PROD)",
+    "changeType": "UPSERT",
+    "aspectName": "dataFlowInfo",
+    "aspect": {
+        "json": {
+            "customProperties": {},
+            "name": "airbyte_workspace"
+        }
+    },
+    "systemMetadata": {
+        "lastObserved": 1690951414157,
+        "runId": "airbyte-test"
+    }
+},
+{
+    "entityType": "dataFlow",
+    "entityUrn": "urn:li:dataFlow:(airbyte,2de688a6-94ee-463a-949b-5a8e808641c0,PROD)",
+    "changeType": "UPSERT",
+    "aspectName": "ownership",
+    "aspect": {
+        "json": {
+            "owners": [],
+            "lastModified": {
+                "time": 0,
+                "actor": "urn:li:corpuser:airbyte"
+            }
+        }
+    },
+    "systemMetadata": {
+        "lastObserved": 1690951414157,
+        "runId": "airbyte-test"
+    }
+},
+{
+    "entityType": "dataFlow",
+    "entityUrn": "urn:li:dataFlow:(airbyte,2de688a6-94ee-463a-949b-5a8e808641c0,PROD)",
+    "changeType": "UPSERT",
+    "aspectName": "globalTags",
+    "aspect": {
+        "json": {
+            "tags": []
+        }
+    },
+    "systemMetadata": {
+        "lastObserved": 1690951414158,
+        "runId": "airbyte-test"
+    }
+},
+{
+    "entityType": "dataJob",
+    "entityUrn": "urn:li:dataJob:(urn:li:dataFlow:(airbyte,2de688a6-94ee-463a-949b-5a8e808641c0,PROD),337a8bf3-230f-4928-bb4d-ed510d7a211b)",
+    "changeType": "UPSERT",
+    "aspectName": "dataJobInfo",
+    "aspect": {
+        "json": {
+            "customProperties": {
+                "status": "'active'",
+                "namespace_definition": "'destination'",
+                "namespace_format": "'${SOURCE_NAMESPACE}'",
+                "prefix": "'oss_'"
+            },
+            "name": "oss_connection",
+            "type": {
+                "string": "COMMAND"
+            }
+        }
+    },
+    "systemMetadata": {
+        "lastObserved": 1690951414159,
+        "runId": "airbyte-test"
+    }
+},
+{
+    "entityType": "dataJob",
+    "entityUrn": "urn:li:dataJob:(urn:li:dataFlow:(airbyte,2de688a6-94ee-463a-949b-5a8e808641c0,PROD),337a8bf3-230f-4928-bb4d-ed510d7a211b)",
+    "changeType": "UPSERT",
+    "aspectName": "dataJobInputOutput",
+    "aspect": {
+        "json": {
+            "inputDatasets": [
```

Comment:
These urns don't look right. It appears that we are creating source and destination datasets corresponding to airbyte connector whereas this shource should really go one level up and create lineage with actual tables that exist in this connectors. An airbyte source or destination is really equivalent to a database or schema container (which may or may not be mapped as DataHub entity) and is not equivalent to what we call a table in postgres/snowflake, etc. I think, the table level information is available in `streams`. Also each airbyte connection will have multiple (source table, destination table) pairs and hence we should use fineGrainedLineages here.